python/schemas/v1/TLSContext.schema

```diff
@@ -34,7 +34,10 @@
         "ecdh_curves": { "type": "array", "items": { "type": "string" }  },
         "secret_namespacing": { "type": "boolean" },
         "redirect_cleartext_from": { "type": "integer" },
-        "sni": { "type": "string" }
+        "sni": { "type": "string" },
+        "verify_certificate_spki": { "type": "array", "items": { "type": "string" } },
+        "verify_certificate_hash": { "type": "array", "items": { "type": "string" } },
+        "verify_subject_alt_name": { "type": "array", "items": { "type": "string" } }
```

Comment:
The `v1` schemas are frozen, so no need to change this -- just leave it as it was. 🙂

Reply:
Done!

Reply:
Also, wasn't sure if you were going to take a crack at the tests; the changes in this PR don't meaningfully test these changes.